package.json

```diff
@@ -1,37 +1,65 @@
 {
-  "name": "frontend-scaffold",
-  "version": "0.1.0",
+  "name": "scaffold",
+  "version": "0.0.1",
   "private": true,
   "scripts": {
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
     "storybook": "start-storybook -p 6006 -c .storybook",
     "test": "jest",
-    "lint": "eslint 'src/**/*.{ts,tsx}'",
-    "pretty": "prettier --check 'src/**/*.{ts,tsx}' --write"
+    "lint": "eslint 'src/**/*.{ts,tsx}' --fix",
+    "pretty": "prettier --check 'src/**/*.{ts,tsx}' --write",
+    "format": "yarn pretty && yarn lint",
+    "cy:run": "yarn run cypress run",
+    "checks:run": "node ./.checks/run.js"
   },
   "dependencies": {
+    "@reach/alert": "^0.10.3",
+    "@reach/checkbox": "^0.10.3",
+    "@reach/dialog": "^0.10.3",
+    "@reach/disclosure": "^0.10.3",
+    "@reach/skip-nav": "^0.10.3",
     "babel-preset-react-app": "^9.1.2",
     "classnames": "^2.2.6",
-    "next": "9.3.6",
+    "framer-motion": "^1.11.0",
+    "lodash": "^4.17.15",
+    "moment": "^2.25.3",
+    "next": "9.4.0",
     "react": "16.13.1",
     "react-dom": "16.13.1",
-    "sass": "^1.26.5"
+    "tailwindcss": "^1.4.6"
   },
   "devDependencies": {
     "@babel/core": "^7.6.4",
-    "@lightbase/eslint-config-lightbase": "^2.2.1",
-    "@storybook/react": "^5.3.18",
+    "@lightbase/eslint-config-lightbase": "^2.11.0",
+    "@storybook/addon-a11y": "^5.3.19",
+    "@storybook/addon-docs": "^5.3.19",
+    "@storybook/addon-knobs": "^5.3.19",
+    "@storybook/preset-typescript": "^3.0.0",
+    "@storybook/react": "^5.3.19",
+    "@testing-library/cypress": "^6.0.0",
+    "@types/classnames": "^2.2.10",
     "@types/jest": "^25.2.1",
+    "@types/lodash": "^4.14.151",
     "@types/node": "^13.13.4",
     "@types/react": "^16.9.5",
+    "@types/react-dom": "^16.9.8",
+    "@types/testing-library__cypress": "^5.0.5",
     "babel-loader": "^8.1.0",
     "babel-plugin-module-resolver": "^4.0.0",
-    "eslint": "^6.8.0",
-    "jest": "^25.1.0",
+    "cypress": "^4.5.0",
+    "eslint": "^7.0.0",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "fork-ts-checker-webpack-plugin": "^4.1.4",
```

Comment:
how do you make this one work without webpack.config? Can't find it in the docs

Reply:
It's part of Storybook see .storybook/main.js

https://github.com/storybookjs/presets/tree/master/packages/preset-typescript

Reply:
Ah, cool, thanks!